packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -745,5 +745,113 @@ describe("SharedString", () => {
             // Verify that the changes were correctly received by the second SharedString
             assert.equal(sharedString2.getText(), "hello friend");
         });
+
+        describe.skip("with interval collection changes", () => {
```

Comment:
can you add a new file for shared interval that we can start using. rather than cramming in shared string

Reply:
or maybe we should jsut rename this file and move the couple of non shared interval things out of it :)

Reply:
we'll let git be the judge of which one is renamed 😛